alwaysontop/alwaysontop.js

```diff
@@ -0,0 +1,41 @@
+const {
+    api,
+    ondblclick,
+    onload,
+    onbeforeunload
+} = window.alwaysOnTop;
+
+
+window.addEventListener('beforeunload', onbeforeunload);
+
+window.addEventListener('dblclick', ondblclick);
+
+document.addEventListener("DOMContentLoaded", () => {
+    onload();
+});
+
+// load all resources from meet
+api._getAlwaysOnTopResources().forEach(src => loadFile(src));
```

Comment:
Mainly. Also this way the toolbar will look exactly like the toolbar in the main window. Also this way we are able to reuse our toolbar implementation from jitsi-meet project.